Dockerfile.cucumber

```diff
@@ -13,4 +13,6 @@ RUN bundle install
 
 RUN mkdir -p /app
 
-COPY . /app
+COPY testing /app
```

Comment:
Turns out I was wrong with these, docker-compose is actually setting the build context to the repository root in all the dockerfiles it references. Interesting, but quite confusing.

You didn't actually have to move the files or change this COPY line. It would appear to be a different issue

Reply:
Interesting! I didn't think that was possible in Docker but I might be misremembering!

Reply:
Yeah, it's docker-compose doing something "helpful". Just running `docker build -f Dockerfile.cucumber` in testing/ dies because Gemfile isn't in the local context.